packages/fake-backends/README.md

```diff
@@ -0,0 +1,8 @@
+# fake-backends
+
+This package contains in-memory implementations for serveral Azure Communication Services backends.
+These fake implementations are intended to support several use-cases independent of actual service backends:
+
```

Comment:
Maybe add a note here about storybook?

Reply:
The first bullet point is about storybook :)